integrations/github/src/haystack_integrations/prompts/github/comment_tool.py

```diff
@@ -0,0 +1,19 @@
+comment_prompt = """
```

Comment:
More just a question, but for for variables like these I believe it's typical to make them in all caps, so COMMENT_PROMPT. What do you think?

Reply:
yes, make sense. 👍

Reply:
Done